### PR TITLE
abstract time crate

### PR DESCRIPTION
**Pull request by sigmaSd**
1 - abstracted `time::Duration`, I think that suffice but I'm not 100% sure.
2 - Change tests accordingly and keep time crate where its useful.
3 - fixed some formatting in doc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +53,7 @@ println!("Current time at UTC: {}", p_tm_utc);
 5- Format the time.
 
 ```rust
-let p_tm = ptime::from_gregorian(time::now());
```
Comment (sigmaSd) [inline README.md:58]: same as `ptime::now()`

**Comment by sigmaSd** [comment]
Can I remove rustc 1.8 from travis ? I don't think its needed plus its blocking the PR.

**Comment by sigmaSd** [comment]
Time create is deprecated, this shouldn't matter anymore

File: README.md (hunk 1 above)
Comment (yaa110) [reply README.md:58]: It is an example of using `from_gregorian`, do not change it

File: README.md [hunk 2]
```diff
@@ -3,23 +3,21 @@ Rust Persian Calendar
...
 ptime = "0.1"
 ```
 
 ## Getting started
-1- Import the crate `ptime`. Most of the time you need to import `time` crate, too.
+1- Import the crate `ptime`.
 
 ```rust
 extern crate ptime;
-extern crate time;
```
Comment (yaa110) [inline README.md:22]: do not remove it

File: src/lib.rs [hunk 3]
```diff
@@ -19,6 +19,7 @@
 //! ```
 
 extern crate time;
+pub use time::Duration;
```
Comment (yaa110) [inline src/lib.rs:22]: do not reexport `Duration`

**Pull request closed without merging**